src/app/pages/viewing/messages/messages.component.ts

```diff
@@ -13,11 +16,50 @@ import { Fonts } from '../../../shared/interfaces';
 export class MessagesComponent implements OnInit {
   fonts: Fonts;
   messageToDelete: string;
-
-  constructor(public viewService: ViewingService, public authService: AuthService) { }
+  constructor(public viewService: ViewingService, public authService: AuthService, private toastr: ToastrService) { }
 
   ngOnInit(): void {
     this.fonts = FontService.fonts;
+    this.viewService.nowMillis = Timestamp.now().toMillis();
+  }
+
+  /**
+   * Generate the lightness value of HSL from RBG
+   */
+  getLightness(r, g, b) {
+    r /= 255, g /= 255, b /= 255;
+    const max = Math.max(r, g, b), min = Math.min(r, g, b);
```

Comment:
Multiple variable declarations per line are not permitted
```suggestion
    const max = Math.max(r, g, b);
    const min = Math.min(r, g, b);
```